### PR TITLE
Rename hidapi-devel to hidapi

### PR DESCRIPTION
**Pull request by chrstphrchvz**
#### Description

As suggested in https://github.com/macports/macports-ports/pull/4142#issuecomment-486049603

Closes #4142

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: comms/hidapi/Portfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+# a6a622f (2016-01-28) is latest commit on master branch as of 2019-04-25;
+# contains macOS-related improvements since 0.8.0-rc1
+github.setup        signal11 hidapi a6a622ffb680c55da0de787ff93b80280498330f
+version             0.8.0
```
Comment (chrstphrchvz) [inline comms/hidapi/Portfile]: I think that the version should be simplified to just `0.8.0`: even though it is prerelease, the upstream project has been updated too infrequently to justify using a more complicated, datestamped version. If and when an actual `0.8.0` release arrives, just increment the revision.

Comment (cjones051073) [reply comms/hidapi/Portfile]: Seems fine to me. I do similar things in other ports.

File: security/libu2f-host/Portfile [hunk 2]
```diff
@@ -27,7 +27,7 @@ checksums           rmd160  eec9e62e6a03348bf5d651bc4a29b112dd9b35be \
 
 depends_build       port:pkgconfig
 depends_lib         port:json-c \
-                    port:hidapi-devel
+                    port:hidapi
```
Comment (l2dy) [inline security/libu2f-host/Portfile]: Revbump needed.

upgrade:
```
--->  Installing hidapi @0.8.0_0
--->  Cleaning hidapi
--->  Unable to deactivate hidapi-devel @0.8.0-20160920_0, the following ports depend on it:
--->  	libu2f-host @1.1.7_0
Warning: Deactivate forced.  Proceeding despite dependencies.
--->  Deactivating hidapi-devel @0.8.0-20160920_0
--->  Cleaning hidapi-devel
--->  Activating hidapi @0.8.0_0
--->  Cleaning hidapi
```
reclaim:
```
--->  Checking for unnecessary unrequested ports
Unrequested ports without requested dependents found: hidapi  @0.8.0_0
Would you like to uninstall them? [Y/n]:
--->  Deactivating hidapi @0.8.0_0
--->  Cleaning hidapi
--->  Uninstalling hidapi @0.8.0_0
--->  Checking for inactive ports
Inactive ports found: hidapi-devel  @0.8.0-20160920_0
Would you like to uninstall them? [Y/n]:
--->  Unable to uninstall hidapi-devel @0.8.0-20160920_0, the following ports depend on it:
--->  	libu2f-host @1.1.7_0
Warning: Uninstall forced.  Proceeding despite dependencies.
--->  Uninstalling hidapi-devel @0.8.0-20160920_0
--->  Building list of distfiles still in use
--->  Searching for unused distfiles
No unused files found.
--->  Scanning binaries for linking errors
--->  Found 2 broken files, matching files to ports
--->  Found 1 broken port, determining rebuild order
You can always run 'port rev-upgrade' again to fix errors.
The following ports will be rebuilt: libu2f-host @1.1.7
```

Comment (chrstphrchvz) [reply security/libu2f-host/Portfile]: Sorry, I wasn't sure whether incrementing the revision was necessary in this case, and neglected to ask. It did not occur to me that already-installed ports "remember" their dependencies even if their portfiles get updated without the version/revision incrementing.

I have opened another PR: #4208

**Pull request merged**